libtiledbvcf/src/write/writer.cc

```diff
@@ -107,15 +108,27 @@ void Writer::init(const IngestionParams& params) {
 
   // Set htslib global config and context based on user passed TileDB config
   // options
-  utils::set_htslib_tiledb_context(params.tiledb_config);
+  std::vector<std::string> vfs_config = params.tiledb_config;
+  try {
+    auto vcf_region = tiledb_config_->get("vcf.s3.region");
+    vfs_config.push_back("vfs.s3.region=" + vcf_region);
+    LOG_INFO("VFS and htslib reading data from S3 region: {}", vcf_region);
+  } catch (...) {
```

Comment:
Do we want to log or anything with the exceptions here?

Reply:
The try/catch here is used to check if `tiledb_config_` contains `vcf.s3.region`, so no action is required if we catch (I added a comment to make that clear). I didn't find a function to check if a config contains a value.

Reply:
Thanks for clarifying. This might be a gap in the c++ api, feel free to open a story (or PR) for this.